.gitignore

```diff
@@ -1,6 +1,197 @@
-/target
-.project
-.settings/**
```

Comment:
Don't really see why we need this fancy .gitignore, but if you want it, at least don't remove existing stuff. This change would make it so the next guy touching the plugin pushes all his local eclipse settings, because they are no longer in the .gitignore